Test/DependencyCollector/FunctionalTests/FuncTest/IIS/IisApplicationPool.cs

```diff
@@ -96,6 +96,7 @@ public IisApplicationPool(string poolName, string managedRuntimeVersion = Defaul
             catch (Exception ex)
             {
                 Trace.TraceError("Exception occured while creating IIS Application Pool: {0}. The exception thrown was {1}", poolName, ex.Message);
+                throw ex;
```

Comment:
This was meant to be a diff PR as this was unrelated. but i did not change branches while making this change.